lib/replace.ts

```diff
@@ -69,7 +77,16 @@ async function replaceJSXTextInFile(
     /* @ts-ignore */
     configFile: false,
   });
-  fs.writeFile(filePath, transformedCode);
```

Comment:
This was the main culprit of test flakiness -- the function wasn't waiting for the file to be written before returning